src/api.ts

```diff
@@ -188,6 +204,18 @@ router.post(
       // Add new bid document to database
       await database.insertBid(newBid);
 
+      const message: BidPlacedMessage = {
+        event: "BidPlaced",
+        version: "1.0",
+        timestamp: new Date().getTime(),
+        logIndex: null,
+        blockNumber: null,
```

Comment:
We need to have these values for tracking historically. Block number tells us which block an event happened on, but there is no guarantee for order of events being read from our queue so we use `logIndex` to track it simply as `0, 1, 2...`

You can get block number using an existing utility in `src > utils > contracts`
```
  const ethersProvider = getEthersProvider();
  const blockNum = ethers.BigNumber.from(await ethersProvider.getBlockNumber());
```
Don't exactly remember how we created logIndex in the past, let me see if I can find reference to this